ecosystem/platform/server/app/controllers/onboarding_controller.rb

```diff
@@ -7,6 +7,7 @@ class OnboardingController < ApplicationController
   before_action :authenticate_user!, except: %i[kyc_callback]
   before_action :ensure_discord!, only: %i[kyc_redirect]
   before_action :ensure_confirmed!, only: %i[kyc_redirect]
+  before_action :ensure_it1_registration_open!, only: %i[kyc_callback kyc_redirect]
```

Comment:
will we have a different flag for node registration, versus KYC?

Reply:
Yes because we intend to close node registration today but leave KYC until Monday.

Reply:
Just to clarify, there are two flags: `:it1_node_registration_disabled` (disables just the node registration step - added in #1111 ) and `:it1_registration_closed` (marks the end of IT1 registration - added in this PR).